backend/src/services/trading_service.py

```
@@ -5,56 +5,99 @@
+        try:
+            payload = {
+                "jsonrpc": "2.0",
+                "id": 1,
+                "method": "getTipFloor",
+                "params": []
+            }
+            # Use the global JITO Block Engine endpoint
+            response = await self.http_client.post("https://mainnet.block-engine.jito.wtf/api/v1/bundles", json=payload)
+            if response.status_code == 200:
+                data = response.json()
+                if "result" in data and data["result"]:
```

Ruff (0.15.13)

[warning] Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)